components/SchoolIcon.tsx

```diff
@@ -0,0 +1,25 @@
+const SchoolIcon: React.FunctionComponent = () => (
```

Comment:
We've moved individual icons into the generic Icon component. I think this re-addition might be a mistake?